huey/consumer.py

```diff
@@ -500,14 +500,16 @@ def start(self):
         # by the child-processes). Once the child processes are created, we
         # restore the handler.
         original_sigint_handler = signal.signal(signal.SIGINT, signal.SIG_IGN)
-        original_sighup_handler = signal.signal(signal.SIGHUP, signal.SIG_IGN)
+        if hasattr(signal, 'SIGHUP'):
```

Comment:
Probably ought to be checking `sys.platform`?

Reply:
I thought about that but I chose this implementation because is more general and is able to manage other platforms that don't support SIGHUP signal.

Let me know if you prefer to user sys.platform to check that and I will make the change

Reply:
Makes sense, thanks!

Reply:
thank you :+1: